### DISCUSSION TITLE
extraParameters setting does not seem to be picked up when running prowin

### DISCUSSION THREAD

**Discussion by RobinJPowell** [vscode-abl/vscode-abl#73] [Q&A] [answered]
I have been getting the addon setup today and was getting "Font nn not in Font Set. (4499)" errors all over the place when I tried to run things or open the AppBuilder.  I have added the extraParameters property to openedge-project.json and set -ininame for the ini file we use, and this has resolved the problem when using "Open file in AppBuilder".  But I still get the errors when using "Run with prowin" to run something up directly from VSCode.  Is there a setting I've missed somewhere to have the parameters apply to both, or is there some other location where it also needs setting?

**Comment by PeterJudgeZA** [comment]
Have you also set `-basekey ini` in the extraParameters?

**Reply by RobinJPowell** [reply]
I had not set that.  I now have, but unfortunatly it has not helped.

**Reply by mikefechner** [reply]
Would be a surprise if that would not be required for AppBuilder - but for executing code. 

-basekey ini is typically only required when the -ininame is progress.ini

**Comment by gquerret** [comment]
Will have a look

**Comment by gquerret** [comment]
There's a difference (unexpected one) when executing the "Open in AB" and "Run in prowin" ; in the former case, extra parameters are after the standard command line parameters, in the latter case, extra parameters are before the standard command line parameters. And the standard parameters include an INI file just to avoid the splash screen (which should in fact be controlled by the option `--dontDelayStartupWithAUselessSplashScreen` and not by an obscure entry in an INI file)

**Reply by gquerret** [reply, marked as answer]
Published 1.4.10 with a fix.

**Reply by RobinJPowell** [reply]
Thanks, that's sorted it.
